package.json

```diff
@@ -84,7 +84,6 @@
     "class-validator": "^0.14.0",
     "compressorjs": "^1.1.1",
     "copy-to-clipboard": "^3.3.3",
-    "core-js": "^3.26.1",
```

Comment:
I'm getting `Cannot find module 'core-js/...` after regenerating yarn.lock after removing explicit core-js dep

Reply:
Strange, the dependency is not used. I have tested these changes in the production and development build processses.

Reply:
require stack:

```
outline-app-1  | Error: Cannot find module 'core-js/modules/es.object.define-property.js'
outline-app-1  | Require stack:
outline-app-1  | - /opt/outline/build/shared/i18n/index.js
outline-app-1  | - /opt/outline/build/server/env.js
outline-app-1  | - /opt/outline/build/server/index.js
outline-app-1  |     at Module._resolveFilename (node:internal/modules/cjs/loader:1047:15)
outline-app-1  |     at Module._load (node:internal/modules/cjs/loader:893:27)
outline-app-1  |     at Module.require (node:internal/modules/cjs/loader:1113:19)
outline-app-1  |     at require (node:internal/modules/cjs/helpers:103:18)
outline-app-1  |     at Object.<anonymous> (/opt/outline/build/shared/i18n/index.js:3:1)
outline-app-1  |     at Module._compile (node:internal/modules/cjs/loader:1226:14)
outline-app-1  |     at Module._extensions..js (node:internal/modules/cjs/loader:1280:10)
outline-app-1  |     at Module.load (node:internal/modules/cjs/loader:1089:32)
outline-app-1  |     at Module._load (node:internal/modules/cjs/loader:930:12)
outline-app-1  |     at Module.require (node:internal/modules/cjs/loader:1113:19) {
outline-app-1  |   code: 'MODULE_NOT_FOUND',
outline-app-1  |   requireStack: [
outline-app-1  |     '/opt/outline/build/shared/i18n/index.js',
outline-app-1  |     '/opt/outline/build/server/env.js',
outline-app-1  |     '/opt/outline/build/server/index.js'
outline-app-1  |   ]
outline-app-1  | }
```

Reply:
my deployment: https://github.com/outline/outline/compare/main...frankli0324:outline:build?expand=1

Reply:
Hmm yes, this doesn't reproduce in development but does in staging. It must be because core-js is a transitive development dependency.

I will restore 👍